docs: update docs for managed identity (#577)
Azure/azure-workload-identity: docs/book/src/topics/federated-identity-credential.md

az ad app federated-credential delete --federated-credential-id $FIC_ID --id $APPLICATION_OBJECT_ID
```

### Federated identity credential for an User-assigned managed identity

```bash

> az identity federated-credential create --name "kubernetes-federated-identity" --identity-name "${USER_ASSIGNED_MANAGED_IDENTITY_NAME}" --resource-group "${RESOURCE_GROUP}" --issuer "${SERVICE_ACCOUNT_ISSUER}" --subject "system:serviceaccount:${SERVICE_ACCOUNT_NAMESPACE}:${SERVICE_ACCOUNT_NAME}"

Reviewer: ```suggestion
az identity federated-credential create \
	--name "kubernetes-federated-identity" \
	--identity-name "${USER_ASSIGNED_MANAGED_IDENTITY_NAME}" \
	--resource-group "${RESOURCE_GROUP}" \
	--issuer "${SERVICE_ACCOUNT_ISSUER}" \
	--subject "system:serviceaccount:${SERVICE_ACCOUNT_NAMESPACE}:${SERVICE_ACCOUNT_NAME}"
```

```

To delete a federated identity credential 

```bash

> az identity federated-credential delete --identity-name "${USER_ASSIGNED_MANAGED_IDENTITY_NAME}" --name "kubernetes-federated-identity" --resource-group "${RESOURCE_GROUP}"

Reviewer: ```suggestion
az identity federated-credential delete \
	--identity-name "${USER_ASSIGNED_MANAGED_IDENTITY_NAME}" \
	--name "kubernetes-federated-identity" \
	--resource-group "${RESOURCE_GROUP}"
```

1. Specify the **Cluster issuer URL**.
1. Specify the **Namespace**.
1. Specify the **Service account name**.

> 1. The **Subject identifier** field autopopulates based on the values you entered.

Reviewer: ```suggestion
1. The **Subject identifier** field auto-populates based on the values you entered.
```

1. Add a **Name** for the federated credential.
1. Click **Add** to configure the federated credential.

![Screenshot showing Azure Portal app registration federated credential screen for Kubernetes scenario](../images/azure-portal-federated-credential-kubernetes.png)

> ### Federated identity credential for an User-assigned managed identity

Reviewer: ```suggestion
### Federated identity credential for a user-assigned managed identity
```

1. Sign in to the [Azure portal](https://portal.azure.com). 
1. Go to resource group and open the user-assigned managed identity you want to configure.
1. Go to **Federated credentials**.
1. In the **Federated credentials** tab, select **Add credential**. The **Add a credential** blade opens.
1. In the **Federated credential scenario** drop-down box select **Kubernetes accessing Azure resources**.
1. Specify the **Cluster issuer URL**.
1. Specify the **Namespace**.
1. Specify the **Service account name**.

> 1. The **Subject identifier** field autopopulates based on the values you entered.

Reviewer: ```suggestion
1. The **Subject identifier** field auto-populates based on the values you entered.
```
